src/Documentation/sidebar.json

```diff
@@ -24,14 +24,42 @@
       {
         "label": "Get Older Files",
         "slug": "older-versions"
+      }
+    ]
+  },
+  {
+    "slug": "tutorials",
+    "source": "tutorials/index.md",
+    "children": [
+      {
+        "slug": "interactive",
+        "label": "Interactive Tutorials"
+      },
+      {
+        "slug": "versioning",
+        "label": "Data Versioning"
+      },
+      {
+        "slug": "pipelines",
+        "label": "Stages and Pipelines"
       },
       {
-        "label": "Example: Versioning",
-        "slug": "example-versioning"
+        "slug": "tutorial",
+        "label": "Longer Tutorial",
```

Comment:
`/doc/tutorials/tutorial/...` seems like a redundant path. How about `tutorials/long`?

Reply:
Maybe "longer" isn't even the best word actually. "Complete"? "Comprehensive"?

Reply:
> Maybe "longer" isn't even the best word actually. "Complete"? "Comprehensive"?

You are right. I was trying to use the same term used here: https://dvc.org/doc/get-started
But I am going to fix it.

Reply:
Cool. Seems like the wording in the Get Started index should be updated too then. Maybe seach any other old mentions to the tutorial? Now /tutorials/tutorial. My main point here was to not use that path though.

Reply:
I've updated the wording and links

Reply:
Oh I see. OK never mind all my recent comments haha.